### DISCUSSION TITLE
How to change the highlighting of terminal-emulator buffer

### DISCUSSION THREAD

**Discussion by wasden** [neovide/neovide#1330] [Q&A]
Neovide works fine for me, but the the highlighting of  terminal-emulator buffer is different from what kconsole shows.
In neovide:
![image](https://user-images.githubusercontent.com/26076025/170197366-d0d67fdf-772c-4426-971c-346ee0b74dfc.png)
In kconsole:
![image](https://user-images.githubusercontent.com/26076025/170198216-0c76a801-7410-4cfe-9307-bb67d884acba.png)

**Comment by AnasR7** [comment]
Just set the color of your highlights in the bash shell or whatever shell you're using right now. not in your konsole term. Bcz terminal-emulator it's set a shell, not the terminal.
